src/jumpstart/saga.ts

```diff
@@ -329,6 +331,17 @@ export function* sendJumpstartTransactions(
 
     ValoraAnalytics.track(JumpstartEvents.jumpstart_send_succeeded, trackedProperties)
     yield* put(depositTransactionSucceeded())
+    yield* put(
+      trackPointsEvent({
+        activityId: 'create-live-link',
+        liveLinkType: 'erc20',
+        beneficiaryAddress,
+        transactionHash: jumpstartTxReceipt.transactionHash,
+        networkId,
+        tokenId: sendToken.tokenId,
+        amount: sendAmount,
```

Comment:
Shouldn't we let the backend retrieve `beneficiaryAddress`, `liveLinkType`, `tokenId` and `amount` from the sent TX?
I know we did something similar for swaps, but I'm slightly worried we'll keep doing this, making it easier for people to abuse the system and send incorrect data.
i.e. we may forget about this and later on we'll start using this unverified data for another purpose while it could be faked.

Reply:
Sorry I should have raised that when we implemented the backend part, similar to this:

https://github.com/valora-inc/points-functions/blob/fd8e68236aee1b1657fcbba2223d9df438a7041b/src/endpoints/trackPointsEvent/types.ts#L26-L33

Reply:
@jeanregisser None of these fields are used for purposes of verification; a user can't abuse the system (i.e., claim points unfairly) by messing with these fields. ATM, these are just used for logging, and `tokenId` is used for eventually rendering the history card (which is the same use case as for swaps). I do understand the concern though, in the case that we _eventually_ use these fields for something more meaningful. I think there are a few options if we want to do something now.

1. Remove all "unnecessary" fields from the request (those not used for rendering the history card), keeping just `tokenId`, and do not bother collecting them in the backend, since they're just for logging.
2. Do the same as 1., but collect those fields from the contract
3. Keep things as-is, but add a comment in the backend (like the one we have for swaps) with the caveat that we would need to collect these data from the transaction on-chain in order to use them for verification.

I kind of lean toward 3., since it works for swaps and I think it's relatively clear that client-provided data shouldn't be trusted. The actual transaction verification is very localized within the code, so it wouldn't be difficult to prevent improper usage with a well-placed comment imo. It's also not clear to me that we have plans in the near future to modify the TX validation requirements.

Reply:
Cool yes 3 sounds good!